netpuppy/__main__.py

```diff
@@ -7,6 +7,12 @@
 import socket
 import sys
 
+def network_port(value):
```

Comment:
just because this project is "trying" to use mypy the function definition should include type hinting

eg
```python
def network_port(value: str) -> int:
```